libexec/swiftenv-install

```diff
@@ -243,6 +247,11 @@ verify=false
 set_global=auto
 set_local=false
 completes=false
+user=true
+
+if [ "$(uname)" == "Darwin" ]; then
+  user=false
```

Comment:
this looks like if we're running on OSX then it defaults `user` to `false`. Don't you only want to do this on Linux?

Reply:
Only macOS supports `--user`, so the default is that user install is disabled. Except on macOS because that was the previous behaviour.

Windows, Linux, *bsd's etc are not using "system" (no-user) installation and the default is `user=true` (--user).